pkg/statistics/handle/ddl/ddl.go

```diff
@@ -116,13 +118,36 @@ func (h *ddlHandlerImpl) HandleDDLEvent(t *util.DDLEvent) error {
 			}
 		}
 	case model.ActionDropTablePartition:
-		// TODO: Update the modify count and count for the global table.
-		_, droppedPartitionInfo := t.GetDropPartitionInfo()
+		globalTableInfo, droppedPartitionInfo := t.GetDropPartitionInfo()
+
+		delta := int64(0)
+		count := int64(0)
```

Comment:
Here it's always `delta = -count`, so I think we can use one variable here.

Reply:
Good cacth!